### ISSUE TITLE
Some folders in datamapper screen not visible after search

### ISSUE THREAD

**Issue by dsimansk** [atlasmap/atlasmap-ui#32]
Original issues https://github.com/syndesisio/syndesis-ui/issues/588 and  https://github.com/syndesisio/syndesis-datamapper-ui/issues/69

I can still reproduce this issue on `syndesis-staging` instance. Is there a specific fix-commit that we could verify that is included in currently deployed version?

Cc @gashcrumb @jbakermd @jknetl

**Comment by jbakermd** [comment]
I've checked this on the latest from the syndesis ui and I can't repro. 

@gashcrumb  might be able to clarify on better setup steps than the following, but here's what I did:

These steps may not apply at all on your staging instance, but if not: 

1. ensure latest syndesis ui code is checked out, run 'git checkout master', then 'git pull'
2. remove the node_modules directory if it exists to be absolutely sure we get the latest datamapper code
3. run 'yarn install'
4. run 'yarn start'

Repro steps in UI

1. In the datamapper UI on the "status" source, expand the top level geoLocation and user folders by clicking on them.
2. Open the search panel for the sources card, type in "lat". Notice that the user folder is hidden, but geolocation is still visible. 
3. Clear the search. All folders (user, geolocation, and others) show back up in the source field list.
